open-api/rest-catalog-open-api.yaml

```diff
@@ -2978,6 +2979,20 @@ components:
           items:
             type: integer
 
+    RemoveSchemasUpdate:
+      allOf:
+        - $ref: '#/components/schemas/BaseUpdate'
+      required:
+        - schema-ids
```

Comment:
Is there a set of updated schema ids or a set of removed schema ids?

Reply:
Yes this is a set of schemas to remove by ID.